### ISSUE TITLE
Passing context changes base font-size on Internet Explorer

### ISSUE THREAD

**Issue by yuderekyu** [dequelabs/axe-core#1585]
### Description

If a context is passed to `axe.run()`, the accessibility check alters the root font size.

I inject axe into an Internet Explorer 11 driver instance (selenium grid 2.53.0). My component has a font-size set to 1.714rem and the base font size is 14px. This translates to a font size of 24px. But, accessibility returns an error stating that the font size is 17.9px, which then fails color contrast ratios for non large fonts. If I do not pass in a context and default to `document`, there are no errors.

Chrome and Firefox do not cause issues. I have updated `axe-core` to the latest release, but the issue still persists.

**Expectation**: `axe.run` to not alter base font-size based on context for supported browsers.

**Actual**: `axe.run`alters base font-size based on context on internet explorer

<pre><code>
axe-core version: 3.0.3
</code></pre>

**Comment by WilcoFiers** [comment]
To the best of my knowledge, axe-core never sets any CSS properties. We'll have to investigate. Do you have a way for us to reproduce the issue? That would help narrow it down.

**Comment by straker** [comment]
Confirmed that when passing a context into axe, IE11 throws a color-contrast issue. https://s.codepen.io/straker/debug/Ezzxbw/xJkjOqJnmKdM

**Comment by straker** [comment]
Found the bug. In IE11, `document` does not have the `msMatchesSelector` function in `element-matches.js` so `color-contrast` errors out when it tries to get all elements on the page starting at `document`. If you look at the results for a no-context run, the `incomplete` section shows the error (not helpful, but it's there).

**Comment by yuderekyu** [comment]
Awesome @straker! I see now I misread `pt` as `px`

@WilcoFiers, I'm sorry I didn't produce a reduce test case in time to speed up this process

**Comment by padmavemulapati** [comment]
Verified the fix availability, Done unit testing. Results are good , no failures.
